keyword_spotting_data_generator/evaluation/extractor/edit_distance_extractor.py

```diff
@@ -34,11 +35,12 @@ def compute_mfccs(self, data):
             fmin=self.f_min,
             fmax=self.f_max)
         data[data > 0] = np.log(data[data > 0])
-        data = [np.matmul(self.dct_filters, x) for x in np.split(data, data.shape[1], axis=1)]
-        return data
+        data = [np.concatenate(np.matmul(self.dct_filters, x)) for x in np.split(data, data.shape[1], axis=1)]
+        return np.array(data)
 
-    def vector_quantization(self, data):
-        raise NotImplementedError
+    def vector_quantization(self, data, code_book):
+        encoding, dist = vq(data, code_book)
```

Comment:
if this is a single line, let's just remove this function, also `vq(data, code_book)[0]` will give you only `encoding`